src/pages/fileCentricCart/utils.js

```diff
@@ -31,12 +31,17 @@ export function convertToCSV(jsonse, comments, keysToInclude, header) {
     let line = '';
     keysToInclude.map((keyName) => {
       if (line !== '') line += ',';
-      line += entry[keyName] !== null ? entry[keyName] : ' ';
+      let columnResult = entry[keyName];
```

Comment:
can we have same logic for the dashboard table csv download too

Reply:
Yes, It is included with https://github.com/CBIIT/bento-frontend/pull/307

Reply:
Okay 👍